.github/workflows/auto-review-and-pr-summary.yml

```
@@ -23,28 +23,31 @@ jobs:
             sleep 30
             
-            # レビューコメントを取得（編集されたコメントも含む）
             COMMENTS=$(gh api repos/$GITHUB_REPOSITORY/issues/$PR_NUMBER/comments)
-          
-            # CodeRabbitのコメントを抽出（キーワードを含んだものを取得）
-            SUMMARY=$(echo "$COMMENTS" | jq -r '.[] | select(.user.login=="coderabbitai[bot]" and (.body | test("<!-- walkthrough_start -->")) and (.body | test("rate limited|review in progress") | not)) | .body')
-
-            echo "--- DEBUG: Checking if statement structure ---"
-            echo 'if [[ -n "$SUMMARY" ]]'
-            echo "---"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)